### ISSUE TITLE
options got reset, now it's not checkingemail at all

### ISSUE THREAD

**Issue by triadne** [inbasic/ignotifier#622]
I don't know if this is related but I wasn't getting notified of an email I had received on my phone, and I have my phone set so it receives emails much slower than this notifier add-on, so I don't have to manually clear them off my phone. 
I checked in the options and it appeared all my inboxes had been cleared. the option to always open the preview window doesn't seem to work either. I'm clicking the addon icon and it's just opening my primary email account. no notification though, so i sent test emails to all my email accounts and I have no notification. I hover over the icon and it shows the accounts there with a 0 next to them 
![image](https://github.com/inbasic/ignotifier/assets/29523389/522350ab-20f1-482e-b9dc-cf367f49be0c)
but I have an email sitting in at least 3 of those accounts. it's been more than 10 minutes now so it seems the add-on is rather busted right now. 

Thanks for working on it, curses to google for messing it up.

**Comment by inbasic** [comment]
do you still have the issue on 1.1.3?

**Comment by inbasic** [comment]
Can you try the notifier on a clean profile to see if you can reproduce this issue?

https://github.com/inbasic/ignotifier/issues/622#issuecomment-1829919070

**Comment by inbasic** [comment]
This means you are looking to check emails of label `tren` on account zero and label `cerw` on account one. If you only want the notifier to check the inbox, use the label `inbox` for the two accounts. If you want the notifier to check all accounts, empty the boxes.

**Comment by triadne** [comment]
Sorry, I don't understand. I guess I have done that wrong then? I thought the label is to put the name on it in the tool tip? If this is all the problem is then firstly I feel foolish, but secondly, I think that section is inherently confusing.

**Comment by inbasic** [comment]
Yes, just remove the `tren` and `cerw` labels and the notifier will work as expected.
